### ISSUE TITLE
Bug: Theano bug when running on Mac

### ISSUE THREAD

**Issue by springcoil** [pymc-devs/pymc#2375]
I was trying to run this https://opendatascience.com/blog/why-hierarchical-models-are-awesome-tricky-and-bayesian/ from Thomas. 
`INFO (theano.gof.compilelock):  ` is the error I get. 
```07/01/2017 

CPython 3.5.2
IPython 5.1.0

numpy 1.11.1
scipy 0.18.1
pandas 0.18.1
matplotlib 1.5.1
seaborn 0.7.0
patsy 0.4.1
pymc3 3.1
theano 0.9.0
joblib 0.10.2

compiler   : GCC 4.2.1 Compatible Apple LLVM 6.0 (clang-600.0.54)
system     : Darwin
release    : 16.6.0
machine    : x86_64
processor  : i386
CPU cores  : 8
interpreter: 64bit
Git hash   : ```

Has anyone seen a bug like this before?

**Comment by springcoil** [comment]
This is really a Theano problem, it seems. https://github.com/pymc-devs/pymc3/issues/1463 looks promising as a solution.

**Comment by springcoil** [comment]
I confirm that the `~/.theanorc` fix mentioned in #1463 fixed this.
